doozer/doozerlib/backend/konflux_client.py

```diff
@@ -331,11 +331,26 @@ async def _get_pipelinerun_template(self, template_url: str):
                 template = jinja2.Template(template_text, autoescape=True)
                 return template
 
+    @staticmethod
+    def _delete_param(params: List, name: str, ):
```

Comment:
I don't think this is required .. we shouldn't create a method for something small like this, which can be done via list comprehension